firmware/application/ui_navigation.cpp

```diff
@@ -554,6 +559,26 @@ void SystemStatusView::rtc_battery_workaround() {
     }
 }
 
+void SystemStatusView::new_sdcard_structure_checker() {
+    const std::filesystem::path root_dir = u"/";
+    const std::filesystem::path resources_dir = u"RES";
+    std::vector<std::filesystem::path> directories = scan_root_directories(root_dir);
+
+    auto scan_result = std::find(directories.begin(), directories.end(), resources_dir);
+
+    if (nav_.is_valid() && (scan_result == directories.end())) {
+        nav_.display_modal(
+            "Warning",
+            "You didn't correctly\n"
```

Comment:
I would propose the following Warning text:

```
SD Card content should be
updated for use with this
firmware version.
See Mayhem Wiki - Update.
```

Personally I would also leave out the grabify link.

Reply:
i was just thinking we need analysis to make sure that users are 90% updated, and when we can remove the checker

Reply:
took the suggestions, fixed

Reply:
ohhh sorry I was just removed grabify, didn't do this, will do it now.